markdown/ranger/ranger-integration-config.html.md.erb

```diff
@@ -118,6 +118,38 @@ Once the connection between HAWQ and Ranger is configured, you can either set up
 6. Select **Service Actions > Restart All** and confirm that you want to restart the HAWQ cluster.
 
 
+## <a id="rpsadminstate"></a>Obtaining the Status of HAWQ/Ranger Integration
+
+Determine the status of HAWQ/Ranger integration in your cluster by:
+
+- Examining the HAWQ packages on your system to verify that you have installed the HAWQ Ranger Plug-in Service. The HAWQ Ranger Plug-in is installed when you install HAWQ.
```

Comment:
I'm not sure if this will work for the apache distribution of 2.2 or not.  Plus - this just shows whether the JARs are installed (not whether connectivity is set-up or has valid connection parameters).  Maybe we should stick to the HAWQ-side commands for checking the auth type.